node/config.go

```diff
@@ -191,6 +191,9 @@ type Config struct {
 	staticNodesWarning     bool
 	trustedNodesWarning    bool
 	oldGethResourceWarning bool
+
+	// Require all transactions to be protected with EIP-155 to prevent replaying on other chains.
```

Comment:
The comment is a bit backwards, since it explains the state when the flag is not set. Imho lets rephrase:

```
// Permits submitting transactions not protected with EIP-155 (unsafe, can be replayed on other chains)
```

Reply:
Also all the other field docs mention the variable, so let's keep it consistent here too:

```
// UnprotectedAllowed permits [...]
```